ghost/i18n/locales/hi/portal.json

```diff
@@ -0,0 +1,60 @@
+    "Account": "khaata",
+    "Account settings": "akaunt seting",
+    "After a free trial ends, you will be charged the regular price for the tier you've chosen. You can always cancel before then.": "shulk pareekshan samaapt hone ke baad, aapake dvaara chune gae star ke lie aapase niyamit mooly liya jaega. aap isase pahale hamesha radd kar sakate hain.",
+    "Already a member?": "kya pahale se hee sadasy hain?",
+    "Back": "peechhe",
+    "Back to Log in": "login par vaapas jaen",
+    "Cancel subscription": "sadasyata radd",
+    "Cancellation reason": "radd karane ka kaaran",
+    "Choose a different plan": "koee bhinn yojana chunen",
+    "Choose your newsletters": "apane nyoozaletars chunen",
+    "Close": "band keejiye",
+    "Comments": "tippaniyaan",
+    "Confirm": "pushti karana",
+    "Continue": "jaaree rakhana",
+    "Delete account": "khaata hata do",
+    "Don't have an account?": "khaata nahin hai?",
+    "Email": "eemel",
+    "Email preference updated.": "eemel vareeyata apadet kee gaee.",
+    "Email preferences": "eemel praathamikataen",
+    "Emails": "eemel praathamikataen",
+    "Emails disabled": "eemel aksham",
+    "Get help": "madad len",
+    "Get notified when someone replies to your comment": "jab koee aapakee tippanee ka javaab de to soochana praapt karen",
+    "Give feedback on this post": "is post par pratikriya den",
+    "Less like this": "is tarah kam",
+    "Manage": "prabandhit karana",
+    "Monthly": "maheene ke",
+    "More like this": "is tarah se adhik",
+    "Name": "naam",
+    "Not receiving emails?": "eemel praapt nahin ho rahe hain?",
+    "Now check your email!": "Now check your email!",
+    "Price": "keemat",
+    "Re-enable emails": "eemel pun: saksham karen",
+    "Retry": "pun: prayaas karen",
+    "Save": "bachaana",
+    "Sending login link...": "login link bheja ja raha hai...",
+    "Sending...": "bhej rahe hein...",
+    "Sign in": "daakhil karana",
+    "Sign up": "sain ap karen",
+    "Start {{amount}}-day free trial": "{{amount}}-din ka nihshulk pareekshan praarambh karen",
+    "Submit feedback": "pratipushti den",
+    "Successfully unsubscribed": "saphalataapoorvak anasabsakraib kiya gaya",
+    "Thanks for the feedback!": "pratikriya ke lie dhanyavaad!",
+    "That didn't go to plan": "vah yojana par nahin gaya",
+    "This site is invite-only, contact the owner for access.": "yah sait keval-aamantrit hai, pahunch ke lie svaamee se sampark karen.",
+    "To complete signup, click the confirmation link in your inbox. If it doesn't arrive within 3 minutes, check your spam folder!": "sainap poora karane ke lie, apane inaboks mein pushtikaran link par klik karen. yadi yah 3 minat ke bheetar nahin aata hai, to apana spaim foldar jaanchen!",
```

Comment:
"~~sainap~~ sign up poora karane ke lie, apane ~~inaboks~~ inbox mein pushtikaran link par ~~klik~~ click karen. yadi yah 3 ~~minat~~ minute ke bheetar nahin aata hai, to apana ~~spaim foldar~~ spam folder jaanchen!",